### ISSUE TITLE
can't zoom any photos on facebook webpage - issue

### ISSUE THREAD

**Issue by lonmaxiz** [extesy/hoverzoom#141]
no picture zoomed and just loading forever. when i hover to any photos on facebook, and i've tried to open it to new tab, it shows the link on address bar like this :

`data:text/html,<img src="https://www.facebook.com/photo/download/?fbid=xxxxxxxxxxxxxxxxxx">`

i'm waiting for so long about the update, i thought it will fixed later. still no improvement update.

i figure it out that facebook now using a new query :
`&ext=147xxxxxxx&hash=AeQT1Ju6oWXGk-Zi`

so, for getting the picture maybe need to add like the query above on src attribute.
`https://www.facebook.com/photo/download/?fbid=xxxxxxxxxxxxxxxxxx&ext=xxxxxxxxxx&hash=?????????????????`

the hash maybe an encryption code. i found in many photos, it seems using only 18 char.
for example of 5 photos : 

```
AeREqHqjwZcDyTva
AeQT1Ju6oWXGk-Zi
AeSx3cUdJ4HNOCiD
AeQdck-nAfZ653v1
AeR5OSSp9wiPhgYI
```

"Ae", the first and second character is seems a constant value.
the third and next after it.
3. R Q S Q R, .... all uppercase alphabetic
4. E T x d 5, .... combination of uppercase, lowercase, and number.
5. still the same like number 4.

i don't know how to trick that. maybe the method should be load the picture page first and get the address.

**Comment by Mte90** [comment]
maybe #275 fix that problem

**Comment by extesy** [comment]
Fixed in `1.0.118`.
